src/components/i18n-helpers.tsx

```diff
@@ -30,3 +32,27 @@ export const useLocaleFormat = (dateFormat) => {
     }
   };
 };
+
+export const useTranslationParagraphs = (
+  ns?: string,
+  options?: UseTranslationOptions,
+) => {
+  const { t, ...useTranslationResponse } = useTranslation(ns, options);
+
+  const tWithParagraphs = (key) => {
+    const texts = t(key).split(/\n\s*\n/);
+
+    return texts.map((text) => {
+      const textWithBreaks = text
+        .split(/\n/)
+        .map((textContent) => [
+          textContent,
+          <br key={textContent.substring(0, 7)} />,
```

Comment:
interesting keying approach. I'm not sure if the key is much better than a plain counter here though. Something you saw somewhere or an idea that you wanted to test? Refreshing to see you solving it like that  for sure ✨